cicecore/drivers/nuopc/cmeps/CICE_RunMod.F90

```diff
@@ -481,9 +508,9 @@ subroutine coupling_prep (iblk)
 
          call ice_timer_start(timer_couple,iblk)   ! atm/ocn coupling
 
-         if (oceanmixed_ice) &
-         call ocean_mixed_layer (dt,iblk) ! ocean surface fluxes and sst
-
+         !if (oceanmixed_ice) &                    Joseph Smith: This has been move to the beginning of the run sequence
```

Comment:
@SmithJos13 Does this change affect other configurations? Probably we could say something when we run UFS WM RTs.